### DISCUSSION TITLE
Filter map by Year

### DISCUSSION THREAD

**Discussion by Biokax** [ppy/osu#14721] [Ideas] [answered]
Idea of filter map by year of ranking would be great for playing old maps without having to create playlist with maps we want.

![image](https://user-images.githubusercontent.com/50368878/132985336-050226d6-5e18-4555-aa71-516df08d85fe.png)

**Comment by peppy** [comment, marked as answer]
For sure. This needs the metadata to be returned for the client, which should happen at some point.
